test/e2e/kogitoapp_test.go

```diff
@@ -119,11 +164,60 @@ func deployKogitoQuarkusExample(t *testing.T, f *framework.Framework, ctx *frame
 					URI:        &gitProjectURI,
 					ContextDir: contextDir,
 				},
-				Native: util.GetBoolEnv("NATIVE"),
+				Native: true,
 			},
 		},
 	}
 
+	deployKogitoServiceApp(t, kogitoService, f, ctx)
+
+	f.Client.Get(goctx.TODO(), types.NamespacedName{Name: kogitoService.Name, Namespace: namespace}, kogitoService)
```

Comment:
All this code can be on another function and be reused for both test cases?

Reply:
Implemented